### PR TITLE
Feature/addwebhook netatmo

### PR DESCRIPTION
**Pull request by sylvainnicolet**
- Remove netatmo module
- Import axios module
- Import express module
- Import qs module
- Create Netatmo class
- Use eslint
- Update README

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+const Netatmo = require('./services/Netatmo')
+const express = require('express');
 
-// Netatmo API Authentication
-let auth = {
-	"client_id": process.env.NETATMO_CLIENT_ID,
-	"client_secret": process.env.NETATMO_CLIENT_SECRET,
-	"username": process.env.NETATMO_USERNAME,
-	"password": process.env.NETATMO_PASSWORD,
+// Netatmo authentification
```
Comment (ralf57) [inline app.js]: `Authentication` instead of `authentification`

File: services/Netatmo.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const util = require('util');
+const qs = require('qs');
+const axios = require('axios');
+const BASE_URL = 'https://api.netatmo.net';
```
Comment (ralf57) [inline services/Netatmo.js]: The BASE_URL could be moved to the `.env` file

**Comment by ralf57** [comment]
@sylvainnicolet  I would reduce the indentation to 2 spaces throughout the whole codebase and, more generally, would be good to integrate https://eslint.org/ into the project so that CS and linting is automatically taken care.

File: services/Netatmo.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		const data = {
+			client_id: args.client_id,
+			client_secret: args.client_secret,
+			username: args.username,
+			password: args.password,
+			scope: args.scope,
+			grant_type: args.grant_type,
+		};
+
+		var url = util.format('%s/oauth2/token', BASE_URL);
```
Comment (ralf57) [inline services/Netatmo.js]: Either use `let` or `const`. `var` is a legacy statement valid for both local and global variables, which can be confusing.

**Comment by ralf57** [comment]
It would be nice to add the command to run the app in the `README.md` file

File: services/Netatmo.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+					headers: {
+						'Authorization': `Bearer ${access_token}`
+					},
+					data: {
+						url: webhook
+					},
+					url,
+				};
+
+				axios(options)
```
Comment (ralf57) [inline services/Netatmo.js]: Why this "sub-call" to the API?

Comment (sylvainnicolet) [reply services/Netatmo.js]: It's to add the webhook URI after we receive the access token

**Pull request merged**